examples/sessions/Elvina/IRC_Test.links

```diff
@@ -117,20 +117,20 @@ fun connectToIRCServer() {
 
     # Test a general message that should be ignored by Links bot.
     writeToSocket("PRIVMSG #irchacks : This is a test message. Please do not reply.\n", rs);
-    print("PRIVMSG #irchacks : This is a test message. Please do not reply.");
+    println("PRIVMSG #irchacks : This is a test message. Please do not reply.");
     readServer(rs, serverTime(), 1);
 
     # End of tests.
     writeToSocket("QUIT\n", rs);
-    print("QUIT");
+    println("QUIT");
     closeSocket(rs);
     ()
 }
 
-# Writes LinksCalc: and given expression to socket; prints it.
+# Writes LinksCalc: and given expression to socket; printlns it.
```

Comment:
your regex should have been something like print[^a-zA-Z] to prevent breaking comments containing "print"

Reply:
Fixed.